<launch>

  <arg name="save_intermediate"           default="false" />
  <arg name="generate_pointclouds"        default="true" />
  <arg name="cleanup"           	  default="false" />
  <arg name="log_to_db"           	  default="true" />
  <arg name="cache_old_data"           	  default="true" />
  <arg name="max_instances"               default="2" />


  <arg name="input_cloud"           	  default="/head_xtion/depth_registered/points" />
  <arg name="input_rgb"           	  default="/head_xtion/rgb/image_color" />
  <arg name="input_depth"           	  default="/head_xtion/depth_registered/image_rect" />
  <arg name="input_caminfo"           	  default="/head_xtion/rgb/camera_info" />

  <node pkg="cloud_merge" type="cloud_merge" name="cloud_merge" output="screen" respawn="true">
	<param name="save_intermediate"  type="bool" value="$(arg save_intermediate)"/>
	<param name="generate_pointclouds"       type="bool" value="$(arg generate_pointclouds)"/>
	<param name="cleanup"  		    	 type="bool" value="$(arg cleanup)"/>
	<param name="log_to_db"  		 type="bool" value="$(arg log_to_db)"/>
	<param name="max_instances"    	 type="int" value="$(arg max_instances)"/>
	<param name="cache_old_data"	type="bool" value="$(arg cache_old_data)"/>
	
	<param name="input_cloud"  		 value="$(arg input_cloud)"/>
	<param name="input_rgb"  		 value="$(arg input_rgb)"/>
	<param name="input_depth"  		 value="$(arg input_depth)"/>
	<param name="input_caminfo"  		 value="$(arg input_caminfo)"/>
  </node>

</launch>
